migration-0.45/dashboards.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var (
+	db *sqlx.DB
+)
+
+type Dashboard struct {
+	Id        int       `json:"id" db:"id"`
+	Uuid      string    `json:"uuid" db:"uuid"`
+	CreatedAt time.Time `json:"created_at" db:"created_at"`
+	CreateBy  *string   `json:"created_by" db:"created_by"`
+	UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
+	UpdateBy  *string   `json:"updated_by" db:"updated_by"`
+	Data      string    `json:"data" db:"data"`
+	Locked    *int      `json:"isLocked" db:"locked"`
```

Comment:
I was referring to this part. It's find since you are not modifying it. There is no need to add these here if they are not part of the select clause.

Reply:
yes not modifying this , only modifying the data, if needed I can remove this as well

Reply:
removed